egs/mgb2_arabic/s5/local/mgb_data_prep.sh

```diff
@@ -39,11 +33,21 @@ head -500 $train_dir/wav_list > $train_dir/wav_list.short
 set -e -o pipefail
 
 xmldir=$db_dir/train/xml/bw
-cat $train_dir/wav_list | while read basename; do
+if [ -z $(which xml) ]; then
```

Comment:
`which` should be avoided, see https://unix.stackexchange.com/questions/85249/why-not-use-which-what-to-use-then for rationale and how to do it correctly

Reply:
OK, thanks. Will fix that.